common/client-core/src/client/real_messages_control/message_handler.rs

```diff
@@ -367,12 +374,12 @@ where
     pub(crate) async fn try_send_reply_chunks(
         &mut self,
         target: AnonymousSenderTag,
-        fragments: Vec<(TransmissionLane, Fragment)>,
+        fragments: Vec<(TransmissionLane, (Fragment, MaxRetransmissions))>,
```

Comment:
nit: that tuple looks quite disgusting tbh. can we perhaps wrap it in some other structure or at least the second part of it(since it's used elsewhere)? idk `FragmentWithRetransmissions`. maybe it should even be 
```rust
struct Foo {
fragments: Vec<Fragment>,
max_retransmissions: MaxRetransmissions
}
```

because realistically you'd want all fragments of your underlying message to have the same retransmitting behaviour

Reply:
Done!